tutorials/iot/raspberrypi-wildlife-camera/raspberrypi-wildlife-camera.md

```diff
@@ -0,0 +1,591 @@
+
+Before starting, make sure you have a GitHub account with an SSH key added to it.
+
+### Push your code to GitHub
+
+Start by creating a [new GitHub repository](https://github.com/new). Pick a name and click "Create repository", do NOT intialize it with a README as we are going to initialize it locally.
+
+-- SCREENSHOT
+
+Now, you need to turn your project folder into a GitHub repository. To do so, run the following commands at the root of your project (make sure you replace "username" and "repository" by the appropriate values):
+
+``` bash
+git init
+git commit -am "Init awesome snapcraft project"
+git remote add origin git@github.com:<username>/<repository>.git
+git push -u origin master
+```
+
+Your repo on GitHub should now contain your project. Make sure it's the case!
+
+### Connect the repo to the build service
+
+Go to [build.snapcraft.io](https;//build.snapcraft.io) and click the "Set up in minutes" button.
+
+You will be prompted to authorize the service to access your GitHub repositories. When done, you will be able to connect your new repository to the build service.
+
+-- SCREENSHOT
+
+Connect it and click the "Not registered" drop down next to your repo name to register your snap on the store. The snap name you are going to use *needs* to match the `name` field of your `snapcraft.yaml` file.
+
+-- SCREENSHOT
+
+### Build and release: this is automatic
+
+The service will automatically start to build your snap for multiple architectures. Note that builds and releases will be triggered each time you do a change on the master branch of the GitHub repository.
+
+When the builds are done, they will be automatically released to the `edge` channel of the store, which means the snap will not appear in store searches and will be invisible to anyone not knowing its exact name.
+
+Upon successful release, you will receive an email on the address connected to your Ubuntu SSO account.
+
+
+After that, it's just a matter of running...
+
+``` bash
+sudo snap install --edge <snap name>
+```
+
+...on any snap-enabled Linux, like your Raspberry Pi!
+
+## Testing on the Pi
+Duration: 5:00
+
+We have a running device, and we have an app to install. It's time to start testing our wildlife camera!
+
+Power on the Pi and plug the USB webcam.
+
+Like before, we are going to access the device with SSH:
+
+``` bash
+ssh username@ip-address
+```
+
+Once connected, install the app. If you have been following the software building steps, you should know the name you have given to it:
+
+``` bash
+snap install <snap name> --edge
+```
+
+Otherwise, install the version this tutorial is based on:
+
+``` bash
+snap install wildlife-camera-tutorial --edge
+```
+
+### Following logs
+
+Since we are going to test it, we want to also see what's happening under the hood, by following the snap output. To do so, open another terminal, connect it to the board with the `ssh` command and run:
```

Comment:
we also want to see...
also, by 'following the snap output' should maybe be 'by monitoring the snap output'